apps/common/hooks/useBalances.tsx

```diff
@@ -63,188 +64,210 @@ const		defaultStatus = {
 	isRefetching: false
 };
 
+async function performCall(
+	ethcallProvider: Provider,
+	calls: Call[],
+	tokens: TUseBalancesTokens[],
+	prices?: TDict<string>
+): Promise<[TDict<TBalanceData>, Error | undefined]> {
+	const	_data: TDict<TBalanceData> = {};
+	const	results = await ethcallProvider.tryAll(calls);
+
+	let		rIndex = 0;
+	for (const element of tokens) {
+		const	{token} = element;
+		const	balanceOf = results[rIndex++] as BigNumber;
+		const	decimals = results[rIndex++] as number;
+		const	rawPrice = formatBN(prices?.[toAddress(token)]);
+		let symbol = results[rIndex++] as string;
+
+		if (toAddress(token) === ETH_TOKEN_ADDRESS) {
+			symbol = 'ETH';
+		}
+		_data[toAddress(token)] = {
+			decimals: Number(decimals),
+			symbol: symbol,
+			raw: balanceOf,
+			rawPrice,
+			normalized: formatToNormalizedValue(balanceOf, Number(decimals)),
+			normalizedPrice: formatToNormalizedValue(rawPrice, 6),
+			normalizedValue: (formatToNormalizedValue(balanceOf, Number(decimals)) * formatToNormalizedValue(rawPrice, 6))
+		};
+	}
+	return [_data, undefined];
+}
+
+async function getBalances(
+	provider: ethers.providers.Web3Provider | ethers.providers.JsonRpcProvider,
+	fallBackProvider: ethers.providers.Web3Provider | ethers.providers.JsonRpcProvider,
+	address: TAddress,
```

Comment:
Follow up on https://github.com/yearn/yearn.fi/pull/72/files#r1099970185

```suggestion
	ownerAddress: TAddress,
```